polaris-core/src/main/java/org/apache/polaris/core/storage/StorageLocation.java

```diff
@@ -19,8 +19,9 @@
 package org.apache.polaris.core.storage;
 
 import jakarta.annotation.Nonnull;
-import java.net.URI;
+import org.apache.hadoop.yarn.logaggregation.LogCLIHelpers;
 import org.apache.polaris.core.storage.azure.AzureLocation;
+import org.apache.hadoop.fs.Path;
 
 /** An abstraction over a storage location */
 public class StorageLocation {
```

Comment:
note, we need tests for the new case being supported

Reply:
Yes, good call out. Included in the latest commit!